docs/advanced-topics/dockstore-cli/dockstore-cli-faq.rst

```diff
@@ -6,10 +6,10 @@ For general FAQs not related to the Dockstore CLI, please see :doc:`our main FAQ
 .. contents:: Table of Contents
   :local:
 
-How does launching CWLs with Dockstore CLI compare with cwltool?
-----------------------------------------------------------------
+How does launching CWLs/WDLs with Dockstore CLI compare with cwltool/Cromwell?
+------------------------------------------------------------------------------
 
-Under the hood, the Dockstore CLI invokes cwltool to launch CWL workflows. However, it adds additional features. The Dockstore CLI can generate JSON parameter files from
+Under the hood, the Dockstore CLI invokes cwltool to launch CWL workflows. Likewise, the Dockstore CLI invokes Cromwell to launch WDL workflows. However, it adds additional features. The Dockstore CLI can generate JSON parameter files from
```

Comment:
The Dockstore CLI invokes cwltool by default, but it is configurable: https://docs.dockstore.org/en/stable/advanced-topics/dockstore-cli/advanced-features.html?highlight=cwl%20runner#alternative-cwl-launchers

There is no such option for WDL, although if/when we add miniwdl support, that will probably change.